source/MaterialXRender/ShaderRenderer.cpp

```diff
@@ -0,0 +1,38 @@
+//
+// TM & (c) 2017 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
+// All rights reserved.  See LICENSE.txt for license.
+//
+
+#include <MaterialXRender/ShaderRenderer.h>
+#include <MaterialXGenShader/HwShaderGenerator.h>
+
+namespace MaterialX
+{
+
+    ImageVec ShaderRenderer::getReferencedImages(const ShaderPtr& shader)
```

Comment:
There is a `generateMipMaps` option which defaults to true. I guess there is no harm in not sending this as an optional argument but once cached if you ask to acquire un-mipmapped version then this won't occur.

Reply:
We can add that option if required. For now it uses default and mimics `bindTextures`